src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -37,6 +37,7 @@ internal class RuntimeMetrics : IDisposable
 #endif
         private static readonly string[] GenNames = new string[] { "gen0", "gen1", "gen2", "loh", "poh" };
         private static readonly int NumberOfGenerations = 3;
+        private static bool isGcInfoAvailable = false;
```

Comment:
nit: You don't really need the `= false` because `false` is the default. If we had them turned on this would be: [CA1805: Do not initialize unnecessarily.](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1805)